torch/onnx/_internal/fx/op_validation.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import annotations
+
+import warnings
+
+from typing import Callable, Dict, List, Sequence, Tuple, Union
+
+import numpy as np
+
+import onnxscript  # type: ignore[import]
+from onnxscript import evaluator  # type: ignore[import]
+
+import torch
+import torch.fx
+
+from torch.onnx import _constants, _type_utils
+from torch.onnx._internal import _beartype, onnx_proto_utils
+from torch.onnx._internal.fx import diagnostics
+from torch.onnx._internal.fx.passes import fx_to_onnxscript
+from torch.utils import _pytree
+
+
+@_beartype.beartype
+def validate_op_between_ort_torch(
+    node: torch.fx.Node,
+    symbolic_fn: Union[onnxscript.OnnxFunction, Callable],
+    torch_args: tuple,
+    torch_kwargs: dict,
+):
+    """Validate the op between ONNX Runtime and PyTorch."""
```

Comment:
It helps to have a few words explaining what to expect would happen.

Reply:
Done